packages/react-core/src/demos/examples/DashboardHeader.js

```diff
@@ -123,20 +124,17 @@ export default class DashboardHeader extends React.Component {
             spacer={{ default: 'spacerNone', md: 'spacerMd' }}
           >
             <ToolbarItem>
-              <Button aria-label="Notifications" variant={ButtonVariant.plain}>
-                <AttentionBellIcon />
-              </Button>
+              <Button aria-label="Notifications" variant={ButtonVariant.plain} icon={<AttentionBellIcon />} />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Button aria-label="App launcher" variant={ButtonVariant.plain} icon={<ThIcon />} />
             </ToolbarItem>
             <ToolbarGroup variant="icon-button-group" visibility={{ default: 'hidden', lg: 'visible' }}>
               <ToolbarItem>
-                <Button aria-label="Settings actions" variant={ButtonVariant.plain}>
-                  <CogIcon />
-                </Button>
+                <Button aria-label="Settings actions" variant={ButtonVariant.plain} icon={<CogIcon />} />
```

Comment:
This is a dropdown in core. I don't know that the "actions" text is helpful either, seems like "Settings" is fine and that's what it is in core.